docs/recipes.rst

```diff
@@ -148,3 +148,29 @@ Also don't forget to rename the folder of your local working copy as Rally will
     mv teamtrackrepo default
 
 From now on, Rally will treat your repository as default and you need to run Rally with ``--track-repository=old-rally-default`` if you want to use the out-of-the-box Rally tracks.
+
+
+.. _recipe_testing_rally_against_ccr_clusters:
+
+Testing Rally against CCR Clusters using a remote metric store
+--------------------------------------------------------------
+
+Testing Rally features (such as the ``ccr-stats`` telemetry device) requiring Elasticsearch clusters configured for `cross-cluster replication <https://www.elastic.co/guide/en/elastic-stack-overview/current/ccr-getting-started.html>`_ can be a time consuming process.
+
+For this reason we've created an example under `recipes/ccr in Rally's repository <https://github.com/elastic/rally/tree/master/recipes/ccr>`_ simplifying it.
```

Comment:
How about something along these lines?

```
Use `recipes/ccr in Rally's repository <https://github.com/elastic/rally/tree/master/recipes/ccr>`_ to test a simple but complete example.
```

(I'd also move this to the previous paragraph)

Reply:
So no need to provide links to cross-cluster-replication from the Elastic website i.e. get rid of the entire line 158 completely, right?

Reply:
No. I'd just turn this into a single paragraph. I don't think that we should have two paragraphs here. So line 158 would stay.

Reply:
Thanks, fixed in https://github.com/elastic/rally/pull/676/commits/7202a0019a53e0220015b8928ef981d823d38241